docs/UserGuide.md

```diff
@@ -424,104 +391,113 @@ Format: **`find`** `FIELD_PREFIX/KEYWORD [FIELD_PREFIX/KEYWORD]…​`
 
 * At least one `KEYWORD` must be provided.
 * The `KEYWORD` is case-insensitive. (e.g `hans` will match `Hans`)
-* Persons matching all keywords will be returned (i.e. `AND` search).
+* People matching all keywords will be returned (i.e. `AND` search).
 
 Examples:
 
 * **`find`** `n/Alex` returns `alex` and `Alexandra`.
-* **`find`** `n/jack tg/M12` returns `Jack` who are from tutorial group `M12`.<br>
+* **`find`** `n/jack tg/M12` returns `Jack` from tutorial group `M12`.<br>
 ![result for 'find n/jack tg/M12'](images/FindJack.png)
 *After execution of Find Command: **`find`** `n/jack tg/M12`*
-* **`find`** `nat/Singaporean` returns all persons who are Singaporeans. <br>
+* **`find`** `nat/Singaporean` returns all Singaporeans. <br>
 ![result for 'find nat/Singaporean'](images/FindSingaporean.png)
 *After execution of Find Command: **`find`** `nat/Singaporean`*
 
-### Sorting all persons: `sort`
+### Sorting people: `sort`
 
-You can sort all persons in the contact book by a specified field.
+You can sort everyone in the contact book by a specified parameter with the `sort` command.
 
-Format: **`sort`** `FIELD_PREFIX/`
+Format: **`sort`** `PREFIX/`
 
-* `FIELD_PREFIX/` can be any of the following: `n/`, `g/`, `p/`, `e/`, `nat/`, `tg/`, `s/`, `r/`, `t/`
-* If field contains words, sort persons in ascending alphabetical order.
-* If field contains numbers, sort persons in ascending numeric order.
+* `PREFIX/` can substituted with any of the following: 
+  * `n/`
+  * `g/`
+  * `p/`
+  * `e/` 
+  * `nat/`
+  * `tg/`
+  * `s/`
+  * `r/` 
+  * `t/`
```

Comment:
Perphaps they could be on a single line as the prefixes are very short

Reply:
Agreed but I prefer that it's standardised with this. Should we put it on a single line?
![image](https://user-images.githubusercontent.com/43673767/140669804-36dfe70c-7dc3-40ed-a1ef-aaab91bb9dfb.png)